hardwareModules/RoFICoM/software/i2c_test/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+cmake_minimum_required(VERSION 3.6)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION OFF)
+set(CMAKE_POSITION_INDEPENDENT_CODE OFF)
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Og")
+set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -Og")
+
+project(i2c_test C CXX ASM)
+
+set(ROFICOM_MCU "STM32G071xx")
+set(ROFICOM_MCU_SPEC "STM32G071GBUx")
+
+file(GLOB_RECURSE SRC src/*.cpp src/*.c)
+file(GLOB_RECURSE LIDAR vl53l1_lidar/*.cpp vl53l1_lidar/*.c)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fmax-errors=5")
+
+# setup_stm32cxx("g0" ${ROFICOM_MCU} ${ROFICOM_MCU_SPEC})
+
+# add_subdirectory( vl53l1_lidar )
+
+add_stm32_target(
+        LIB
+        TARGET vl53l1_lidar
+        LIBTYPE STATIC
+        MCU ${ROFICOM_MCU}
+        MCU_SPEC ${ROFICOM_MCU_SPEC}
+        LINKER_SCRIPT $ENV{ROFI_ROOT}/hardwareModules/RoFICoM/software/i2c_test/stm32/STM32G071GBUx_FLASH.ld
+        FILES ${LIDAR})
+target_include_directories(vl53l1_lidar PUBLIC
+        vl53l1_lidar/
+        vl53l1_lidar/core/inc
+        vl53l1_lidar/platform/inc)
+target_link_libraries(vl53l1_lidar
+        g0_stm32cxx_i2c g0_stm32cxx_gpio
+        atoms)
+target_compile_definitions(vl53l1_lidar PUBLIC VL53L1_DEBUG=1 PAL_EXTENDED=1)
```

Comment:
(není to doporučení, jen otázka): Nedávalo by smysl DEBUG flag nastavovat podle typu buildu?